client/components/orderable-list/test/payment-method.js

```diff
@@ -19,33 +20,33 @@ describe( 'PaymentMethod', () => {
 		expect( screen.getByText( 'Bar' ) ).toBeInTheDocument();
 	} );
 
-	test( 'renders Manage and Delete buttons', () => {
+	test( 'renders "Manage" and "Delete"', () => {
 		render( <PaymentMethod label="Foo" /> );
 
-		const manageButton = screen.getByRole( 'button', {
+		const manageLink = screen.getByRole( 'link', {
```

Comment:
Thanks for pointing this out! I did notice this but I was unsure how to make it clear what a test is doing so just for clarity's sake, I went for the redundant checks, otherwise there'd just be the `get*` call with no explanation what is being tested.

What would be the best way to approach this? Change the `get*` calls to `query*`?